Source/Shared/Cache/BasicHybridCache.swift

```diff
@@ -34,9 +34,16 @@ public class BasicHybridCache {
   }
 }
 
-// MARK: - Data protection
+// MARK: - Disk cache
 
 public extension BasicHybridCache {
+  /**
+   Calculates total disk cache size
+   */
+  func totalDiskSize() throws -> UInt64 {
```

Comment:
Just read the issue and that is somebody that needs it. I thought we need it for something